custom_components/afvalwijzer/sensor.py

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
You are missing `asyncio `to be able to  use `wait`
```suggestion
import asyncio
```